File: core/src/test/java/ml/glassmc/loader/test/RunHooksTest.java
package ml.glassmc.loader.test;

import ml.glassmc.loader.GlassLoader;
import ml.glassmc.loader.test.shard.TestHook;
import org.junit.jupiter.api.Test;

public class RunHooksTest {

    @Test
    public void runHooksTest() {
        GlassLoader.getInstance().runHooks(TestHook.class);
    }

}

File: core/src/test/kotlin/ml/glassmc/loader/test/RunHooksTest.kt
package ml.glassmc.loader.test

import ml.glassmc.loader.GlassLoader
import ml.glassmc.loader.test.shard.TestHook
import org.junit.jupiter.api.Test

class RunHooksTest {

    @Test
    fun test() {
        GlassLoader.runHooks(TestHook::class.java)
    }

}